code-reviews/example3/adventure_game.py

```diff
@@ -0,0 +1,79 @@
+# An interactive adventure game!
+import random
+
```

Comment:
Would you add a comment showing you are defining functions?

Reply:
Yes good call!